internal/api/httpsrv/handler.go

```diff
@@ -0,0 +1,127 @@
+package httpsrv
+
+import (
+	"context"
+
+	"github.com/gin-gonic/gin"
+	"github.com/google/uuid"
+
+	"go.infratographer.com/identity-manager-sts/internal/storage"
+	"go.infratographer.com/identity-manager-sts/internal/types"
+	v1 "go.infratographer.com/identity-manager-sts/pkg/api/v1"
+)
+
+// apiHandler represents an API handler.
+type apiHandler struct {
+	engine storage.Engine
+}
+
+func (h *apiHandler) CreateIssuer(ctx context.Context, req CreateIssuerRequestObject) (CreateIssuerResponseObject, error) {
+	createOp := req.Body
+
+	var (
+		claimsMapping types.ClaimsMapping
+		err           error
+	)
+
+	if createOp.ClaimMappings != nil {
```

Comment:
This will error out if `createOp` is nil.

Reply:
That's true. However, if `req.Body` is nil that means we somehow got a `CreateIssuerRequestObject` with a nil body, which means that oapi-codegen's [strict server generation][oapi-strict] is broken. If our request validation is broken, I'd rather it break loudly.

[oapi-strict]: https://github.com/deepmap/oapi-codegen#strict-server-generation